src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/http_action/whoami/WhoAmIRequest.java

```diff
@@ -15,7 +15,13 @@
 
 package com.amazon.opendistro.elasticsearch.performanceanalyzer.http_action.whoami;
 
+import java.io.IOException;
 import org.elasticsearch.action.support.nodes.BaseNodesRequest;
+import org.elasticsearch.common.io.stream.StreamInput;
 
 public class WhoAmIRequest extends BaseNodesRequest<WhoAmIRequest> {
+  // for serialization
```

Comment:
nit: you can remove this comment, or add more info about why we need this for serialization.

Reply:
comment removed